sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,649 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["MBristle"]
+
+import logging
+from typing import Union
+
+import pandas as pd
+from pandas.api.types import is_integer_dtype
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+
+class AutoTSAdapter(BaseForecaster):
+    """Act as adapter for the AutoTS library.
+
+    Parameter:
+    ---------
```

Comment:
You have indented all contents of `Parameter` section. This is not in line with Numpydoc, so Read the docs is showing an additional quote style. Can you please remove the extra indentation?

Compare these two pages:

https://sktime--5948.org.readthedocs.build/en/5948/api_reference/auto_generated/sktime.forecasting.adapters.AutoTSAdapter.html

vs

https://sktime--5948.org.readthedocs.build/en/5948/api_reference/auto_generated/sktime.forecasting.model_selection.ForecastingGridSearchCV.html

Reply:
I think there are also some line break related issues, but I am ignoring those for the time being.

Reply:
Comments are still not finished.. sorry! I had them in a different quote style and had to rewrite them. Also, I'm currently copy-pasting the original ones.. Still a todo